src/js/utils/display-name.js

```diff
@@ -24,6 +24,11 @@ const findInvalidCharacters = (displayName) => {
 	const containsCharactersNotInCoralTalkRules = /[^a-z0-9_.]/gi;
 	const matchingCharacters = displayName
 		.match(containsCharactersNotInCoralTalkRules);
+
+	if (!matchingCharacters) {
```

Comment:
Can we just include this in the ternary below? I have added a suggestion

Reply:
Suggestion committed.